src/cli/java/org/commcare/util/screen/QueryScreen.java

```diff
@@ -62,6 +62,8 @@ public class QueryScreen extends Screen {
 
     private boolean defaultSearch;
 
+    private boolean dynamicSearch;
```

Comment:
How does this get set for `QueryScreen` ?

Reply:
https://github.com/dimagi/commcare-core/pull/1358/commits/8bfbc6f54aa0b5eb32a1c7c493cafa7022cb0390